### DISCUSSION TITLE
posts/how-to-create-a-blog-post

### DISCUSSION THREAD

**Discussion by giscus** [dgelineau/next-mantine-blog-template#14] [Comments]
# posts/how-to-create-a-blog-post

This is a guide on how to create a blog post

https://next-mantine-blog-template.vercel.app/posts/how-to-create-a-blog-post

**Comment by dgelineau** [comment]
Here's our first comment! :smile:

```tsx
function Video({ video }) {
  return (
    <div>
      <Thumbnail video={video} />
      <a href={video.url}>
        <h3>{video.title}</h3>
        <p>{video.description}</p>
      </a>
      <LikeButton video={video} />
    </div>
  );
}
```

| Parameter | Type     | Description                         |
| :-------- | :------- | :---------------------------------- |
| `name`    | `string` | **Required**. Name of the component |
